package.json

```diff
@@ -37,22 +37,9 @@
     "watch": "npm run watch:js | npm run watch:css",
     "watch:js": "babel --config-file ./config/babel.config.json --watch src --out-dir dist",
     "watch:css": "postcss --config config/postcss.config.js --watch src/**/*.css --base src --dir dist",
-    "test:app:start": "docker-compose -f test/docker/app.yml up -d",
-    "test:app:rebuild": "npm run build && docker-compose -f test/docker/app.yml up -d --build",
-    "test:app:stop": "docker-compose -f test/docker/app.yml down",
-    "test": "npm run test:app:rebuild && npm run test:integration && npm run test:app:stop && npm run test:types",
-    "test:db": "npm run test:db:mysql && npm run test:db:postgres && npm run test:db:mongodb && npm run test:db:mssql",
-    "test:db:mysql": "node test/mysql.js",
-    "test:db:postgres": "node test/postgres.js",
-    "test:db:mongodb": "node test/mongodb.js",
-    "test:db:mssql": "node test/mssql.js",
-    "test:integration": "mocha test/integration",
+    "test": "echo \"Write some tests...\"; npm run test:types",
```

Comment:
😢 , I was thinking about this, though. 

The best IMO would be to have integration/e2e tests covering the sig-in/sign-out user paths provided by **NextAuth**. 

However, I think we could still even benefit from unit tests, and they could serve as documentation as well. We could, for instance, aim to have all the exports of a given module unit tested. 🤔

Reply:
Yes, we have options how to create some basic but still useful tests that could benefit us, and would like to chat about them with @iaincollins.